src/main/java/org/folio/innreach/domain/service/impl/VirtualRecordServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.innreach.domain.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.innreach.domain.service.HoldingsService;
+import org.folio.innreach.domain.service.InstanceService;
+import org.folio.innreach.domain.service.ItemService;
+import org.folio.innreach.domain.service.LoanService;
+import org.folio.innreach.domain.service.VirtualRecordService;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class VirtualRecordServiceImpl implements VirtualRecordService {
+
+  private final ItemService itemService;
+  private final HoldingsService holdingsService;
+  private final InstanceService instanceService;
+  private final LoanService loanService;
+  @Override
+  public void deleteVirtualRecords(UUID folioItemId, UUID folioHoldingId, UUID folioInstanceId, UUID folioLoanId) {
+    log.debug("deleteVirtualRecords :: parameters folioItemId:{} folioHoldingId:{} folioInstanceId:{} folioLoanId:{}",folioItemId,folioHoldingId,folioInstanceId,folioLoanId);
```

Comment:
Line is a bit long and needs spaces. Maybe break after the formatted string and put the params on a new line with spaces between them.

Reply:
This line would be more readable as (notice the new line/break). Also add spaces between params:
```
log.debug("deleteVirtualRecords :: parameters folioItemId: {} folioHoldingId: {} folioInstanceId: {} folioLoanId: {}",
  folioItemId, folioHoldingId, folioInstanceId, folioLoanId);
```